docs/common/registration.md

```diff
@@ -55,7 +55,7 @@ OSG resources are stored under a hierarchy of facilities, sites, and resource gr
 |----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Facility       | The institution or company where your resource is located, e.g. `University of Wisconsin`                                                                              |
 | Site           | Smaller than a facility; typically represents an academic department, research group, or a computing center, e.g. `CHTC` for the Center for High Throughput Computing. |
-| Resource Group | A logical grouping of resources, e.g. you may group together resources that serve your Slurm cluster under a resource group named `CHTC-Slurm-HPC`                     |
+| Resource Group | A logical grouping of resources, e.g. you may group together resources for testing under a resource group named `CHTC-ITB` and by setting `Production: False`          |
```

Comment:
I don't think this is the right place for `Production: False`. Perhaps do something like:
"A logical grouping of resources. Production and testing resources should be placed into separate Resource Groups"
and then have the Production setting explained later.